### ISSUE TITLE
UI Suggestion for v3. Put the transient menu and horizon view in the clarify screen. 

### ISSUE THREAD

**Issue by maikol-solis** [Trevoke/org-gtd.el#143]
Currently, if you try to clarify something, you see this screen 

![image](https://user-images.githubusercontent.com/4078489/236694981-0ff727db-57e2-4b1b-a673-c5a288cf6af6.png)

However, I feel that a better UI could be something like this since the beginning:

![image](https://user-images.githubusercontent.com/4078489/236695479-690df681-4e32-4435-a713-b10c21e827b0.png)

Here, you have everything ready to start handling your tasks. 

Otherwise, you'll need to assign `org-gtd-organize` and `org-gtd-clarify-toggle-horizon-window` to some keybindings or call them manually each time. 

WDYT?

**Comment by Trevoke** [comment]
Well, there's a few things here.

First, regarding the `horizons` window, there is a custom variable you can set to have it show by default.

Second, regarding the transient menu, I'm not actually sure how to give control to another window while it's up, so invoking it right away would probably mean the user has to close it anyway (if you know how to clarify your item while the menu is up, let me know!)

Third, this is not great for people working on possibly small monitors, so I'd rather leave as much as possible here to configuration, like displaying the horizons buffer by default.

And, finally, the `org-gtd-clarify-map` exists precisely for the sake of binding functions to keyboard shortcuts in this particular step of the process, including for instance the new function `org-gtd-clarify-project-insert-template`, still obviously undocumented as well, but requested in #121 

You sound like you hesitate or don't want to use keybindings here, would you help me understand why that is?

**Comment by maikol-solis** [comment]
Yes. I wasn't aware of `org-gtd-clarify-show-horizons` at all. Thanks for pointing out. 

Just a little remark here. In the variable docstring 

https://github.com/Trevoke/org-gtd.el/blob/ba42650860ed087ca9d0eb10c3459dd7a281ac2f/org-gtd-clarify.el#L38-L44

The non-nil (t) value causes an error. It should be the position where you want the popup-buffer (right, left, top or bottom). I set it with this line: 

```
  (setq org-gtd-clarify-show-horizons 'right)
```

Now, the transient menu it's just thinking about having the options right away when you start the clarifying process. But it's optional at the end. I have `org-gtd-clarify` bound to `C-c C-c` for easy ergonomics.

**Comment by maikol-solis** [comment]
Well, in the end, it's just giving the users all the options right away. 

But it's your call, of course. 

Thanks.

**Comment by Trevoke** [comment]
What I mean is -- please try to open the `organize` menu and then try to
clarify the item. If you manage, let me know how you did it, please.

**Comment by Trevoke** [comment]
I don't mind _somehow_ telling the user what all the options are, I think
that's probably a good idea, and it might even be worth doing before they
start clarifying, but I don't think it can be done with the transient menu.
